lib/github-pages.rb

```diff
@@ -7,29 +7,31 @@ class GitHubPages
   def self.gems
     {
       # Jekyll
-      "jekyll"                => "2.4.0",
-      "jekyll-coffeescript"   => "1.0.1",
+      "jekyll"                => "3.0.0",
       "jekyll-sass-converter" => "1.3.0",
```

Comment:
Since jekyll-coffeescript was moved to plugins, shouldn't jekyll-sass-converter be moved as well?

And are the plugins listed in a particular order?